disperser/cmd/apiserver/flags/flags.go

```diff
@@ -150,7 +150,7 @@ var (
 	MaxNumSymbolsPerBlob = cli.UintFlag{
 		Name:     common.PrefixFlag(FlagPrefix, "max-num-symbols-per-blob"),
 		Usage:    "max number of symbols per blob. This flag is only relevant in v2",
-		Value:    65_536,
+		Value:    524_288, // this should allow for 16MiB blobs
```

Comment:
Nit, would it make sense to express this as a mathematical expression? I verified that your number made sense, but I personally find it easier to do when the parts are split out. Feel free to ignore suggestion if you don't like it.

```suggestion
		Value:    1024 * 1024 * 16 / encoding.BYTES_PER_SYMBOL, // this should allow for 16MiB blobs
```